MekHQ/src/mekhq/campaign/personnel/enums/PersonnelStatus.java

```diff
@@ -224,6 +246,38 @@ public static PersonnelStatus parseFromString(final String text) {
                     return STUDENT;
                 case 5:
                     return MISSING;
+                case 6:
+                    return POW;
+                case 7:
+                    return ON_LEAVE;
+                case 8:
+                    return AWOL;
+                case 9:
+                    return RESIGNED;
+                case 10:
+                    return DESERTED;
+                case 11:
+                    return DEFECTED;
+                case 12:
+                    return HOMICIDE;
+                case 13:
+                    return WOUNDS;
+                case 14:
+                    return DISEASE;
+                case 15:
+                    return ACCIDENTAL;
+                case 16:
+                    return NATURAL_CAUSES;
+                case 17:
+                    return OLD_AGE;
+                case 18:
+                    return MEDICAL_COMPLICATIONS;
+                case 19:
+                    return PREGNANCY_COMPLICATIONS;
+                case 20:
+                    return UNDETERMINED;
+                case 21:
+                    return SUICIDE;
```

Comment:
ever play Rimworld? There's also FOOD_BINGE and TANTRUM
:D

Reply:
Ironically, given I've become the MechSims guy, I don't actually _like_ playing games like Rimworld or Dwarf Fortress. Though I do enjoy listening to people recount their Dwarf Fortress misadventures.